pages/docs/reference/server-overview.md

```diff
@@ -36,6 +36,8 @@ for Kotlin, including [full documentation](http://vertx.io/docs/vertx-core/kotli
 It serves as an alternative to traditional templating systems such as JSP and FreeMarker.
 
  * [Micronaut](https://micronaut.io/) is a modern, JVM-based, full-stack framework for building modular, easily testable microservice and serverless applications. It comes with a lot of built-in, handy features.
+ 
+ * [Javalin](https://javalin.io) is a very lightweight web framework for Kotlin and Java which supports WebSockets, HTTP2 and async requests. Javalin’s main goals are simplicity, a great developer experience, and first class interoperability between Kotlin and Java.
```

Comment:
Please remove the last sentence.

Reply:
Thank you. I am not related to Javalin developers. But I think it still worth it to be included on this page.

Reply:
Thanks!
We want to publish more info about Kotlin for server side on a dedicated section of the website later. This will include details about available frameworks: features, concepts, examples, etc.
For now, we're just gathering the list of frameworks worth looking at, without details or recommendations. Thanks for helping with it!